tests/unit/backend/fixtures/mock_aws_test_case.py

```diff
@@ -116,21 +114,21 @@ def generate_dataset_with_s3_resources(self, session, artifacts=True, explorer_s
             self.create_explorer_s3_object(session, dataset.id, upload=True)
         return dataset
 
-    def get_s3_object_paths_from_dataset(self, dataset: Dataset) -> typing.List[typing.Tuple[Bucket, str]]:
+    def get_s3_object_paths_from_dataset(self, dataset: Dataset):
```

Comment:
There doesn't seem to be a good/easy way to import the `Bucket` class using boto3 (as the class doesn't seem to be publicly exposed) so I am just getting rid of the type annotations here.